<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Optimist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pessimist</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pignistic</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>557</y>
     <width>330</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="LeftSwitch">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>◃</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RightSwitch">
        <property name="text">
         <string>▹</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>90</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Combination rule :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Filter's parametres :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>130</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Decision :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Matrix 3*3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Matrix 5*5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Matrix 7*7</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dempster Shafter</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yager</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Disjunctive</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dubois Parade</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PRC5</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="PreviewVar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>189</y>
     <width>311</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border-radius:0px;
border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="result">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>234</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>234</width>
     <height>50</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	text-align:left;
	color:#fff;
	border:0px;
	border-radius:20px;
	font-size:15px;
	cursor: pointer;
	background-color:#538fff;
}
QPushButton:hover{
background-color:#3668ff;
}</string>
   </property>
   <property name="text">
    <string>Result</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>assets/img/path395.png</normaloff>assets/img/path395.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>169</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
      </property>
      <property name="text">
       <string>Preview</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

color:#4b6272;
border:0px;
font-size:15px;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>Age :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="age">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="file.qrc"/>
 </resources>
 <connections/>
</ui>
